sceptre/synapseprod/templates/snowflake-access.yaml

```diff
@@ -0,0 +1,61 @@
+Description: Setup cross account IAM access.  Give user in another AWS account access to resources.
```

Comment:
is this setup from snowflake docs?  if so it might be good to add a comment in this file referencing the doc.

Reply:
Done.